tests/src/Kernel/Access/WikiTest.php

```diff
@@ -204,15 +225,6 @@ public function entityOperationAccessProvider() {
         ],
       ],
       [
-        // Non-members should have the right to edit any group content.
```

Comment:
@amitaibu Not just now. I removed it because the old test assigned a non-member role to a membership. The member role is assigned to a membership automatically so that's not a problem here.

I mentioned some time ago I didn't know how to make the Wiki functionality work for a core authenticated user. Once I understand how the wiki works (i.e. what enables wiki on a piece of content) I can write a test for the "non-member" role - a core authenticated use r who isn't a member of the group.